cmd/nodecmd/local.go

```diff
@@ -415,14 +416,17 @@ func localStopNode(_ *cobra.Command, _ []string) error {
 func localDestroyNode(_ *cobra.Command, args []string) error {
 	clusterName := args[0]
 
-	localStopNode(nil, nil)
+	if err := localStopNode(nil, nil); err != nil {
+		return fmt.Errorf("failed to destroy local node: %w", err)
```

Comment:
node can be already stopped. for the moment on ignore the error or verify is the node is already stopped,
we want to continue removing all the directorioes that is the reason the error is ignored